src/pages/PMT_PaymentWizard.page

```diff
@@ -222,11 +222,12 @@ Copyright (c) 2008 Evan Callahan, evanc@npowerseattle.org, NPower Seattle, 403 2
                     <table class="slds-table" >
                         <thead>
                             <tr class="slds-text-heading--label">
-                                <th scope="col"><span class="slds-truncate">{!$Label.pmtWizardPaymentNumber}</span></th>
-                                <th scope="col"><span class="slds-truncate">{!$ObjectType.npe01__OppPayment__c.Fields.npe01__Payment_Amount__c.Label}</span></th>
-                                <th scope="col"><span class="slds-truncate">{!$ObjectType.npe01__OppPayment__c.Fields.npe01__Scheduled_Date__c.Label}</span></th>
-                                <th scope="col"><span class="slds-truncate">{!$ObjectType.npe01__OppPayment__c.Fields.npe01__Payment_Date__c.Label}</span></th>
-                                <th scope="col"><span class="slds-truncate">{!$ObjectType.npe01__OppPayment__c.Fields.npe01__Paid__c.Label}</span></th>
+                                <th scope="col" class="slds-cell-wrap">{!$Label.pmtWizardPaymentNumber}</th>
+                                <apex:repeat value="{!$ObjectType.npe01__OppPayment__c.FieldSets.Payment_Wizard}" var="field">
+                                     <th scope="col" class="slds-cell-wrap"><span class="slds-truncate">
+                                        <apex:outputText value="{!IF($ObjectType.npe01__OppPayment__c.fields[field].type == 'reference', $ObjectType.Contact.fields[field].relationshipName, field.label)}"/>
```

Comment:
I feel like I did this for a reason but will put it to just label and give it a try.

Reply:
LOL - I obvs copied this pattern from elsewhere and didn't test, because the much graver error here is that I'm trying to use the relationship name from Contact! `$ObjectType.Contact.fields[field].relationshipName` 
Switching to Label as you discuss.